packages/template-retail-react-app/README.md

```diff
@@ -6,10 +6,8 @@ Developers don’t have to worry about the underlying infrastructure, whether th
 
 ## Requirements
 
-```
-  Node ^12.x or ^14.x
-  npm ^6.14.4
-```
+-   Node ^14.x
+-   npm ^6.14.4
```

Comment:
```suggestion
-   Node 14
-   npm 6.14.4 or newer
```
Do we expect semver range specifiers (the leading `^`) to be widely understood? Also, we have npm 7 and 8 in our CI pipeline, do we officially support them?

Reply:
Yeah, let's ditch the carret and account for newer versions of npm. BTW, Salesforce style uses "or later" instead of "or newer." I'll update this in the main readme too.